jamon-parent/jamon-examples/pom.xml

```diff
@@ -0,0 +1,32 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.wicketstuff</groupId>
+		<artifactId>wicketstuff-jamon-parent</artifactId>
+		<version>7.2.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>jamon-examples</artifactId>
+	<name>Jamon examples</name>
+	<description>Example usage of Wicketstuff Jamon</description>
+
```

Comment:
Please add the properties for disabling the deployment, javadoc, etc.
Consult with the other -examples/pom.xml.